src/renderer/screens/ColorPalette/index.js

```diff
@@ -0,0 +1,184 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2018, 2019  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This is Reactjs functional component that create palette for change buttons color on keyboard
+ */
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import ColorButtonsArea from "./ColorButtonsArea";
+import PickerColorButton from "./PickerColorButton";
+
+ColorPalette.propTypes = {
+  classes: PropTypes.object.isRequired,
+  onColorSelect: PropTypes.func.isRequired,
+  palette: PropTypes.array.isRequired,
+  onColorPick: PropTypes.func.isRequired,
+  disabled: PropTypes.bool.isRequired,
+  selected: PropTypes.number.isRequired
+};
+
+const styles = theme => ({
+  root: {
+    display: "flex",
+    position: "fixed",
+    left: 0,
+    right: 0,
+    bottom: 0,
+    height: 150,
+    justifyContent: "center",
+    alignItems: "center"
+  },
+  palette: {
+    display: "flex",
+    height: 70,
```

Comment:
Could we enlarge this to 80, please? See larger comment about resizing this entire palette.

Reply:
@TreTuna 

Sorruy, Its my fault
In `ColorPalette.js` there were makes changes to dinamic height depends of the width of screenSize - it will makes height of colorPalette more flexible, but didn't set `minHeight` yet. - my fault. Will update for today.